addons/mrp/views/mrp_workcenter_views.xml

```diff
@@ -209,7 +209,7 @@
                                         <button t-if="record.workorder_count.raw_value &gt; 0" class="btn btn-primary" name="action_work_order" type="object" context="{'search_default_ready': 1, 'search_default_progress': 1, 'desktop_list_view': 1, 'search_default_workcenter_id': id}">
                                             <span>WORK ORDERS</span>
                                         </button>
-                                        <button t-if="record.workorder_count.raw_value &lt;= 0" class="btn btn-warning" name="%(act_product_mrp_production_workcenter)d" type="action">
+                                        <button t-if="record.workorder_count.raw_value &lt;= 0" class="btn btn-warning" name="%(mrp.mrp_workorder_todo)d" type="action">
```

Comment:
Because it's stable, no touching the action record itself, but you may want to delete the record in the master PR since it is the only place it was called.

Reply:
Can you elaborate on this a bit? You mean I should implement the logic in the existing action instead of writing a new one?

Reply:
what I mean:
- this commit for 18.0 is ok, no need to change anything
- make a new PR in master to remove the record of `act_product_mrp_production_workcenter` + the mrp.workorder dependent record + an upgrade script because it's the only place it is called, for cleanup